Python/bytecodes.c

```diff
@@ -4916,6 +4916,12 @@ dummy_func(
             DEOPT_IF(func->func_version != func_version);
         }
 
+        tier2 op(_CHECK_FUNCTION_INLINE, (func_version/2, callable_p/4 --)) {
```

Comment:
```suggestion
        tier2 op(_CHECK_FUNCTION_UNMODIFIED, (func_version/2, callable_p/4 --)) {
```

The problem with "INLINE" is that doesn't what is inlined and into what.